### PR TITLE
feat: aptabase 0.0.5

### PR DESCRIPTION
**Pull request by herefishyfish**
Adds basic android support for Aptabase.

### CODE REVIEW

**Comment by herefishyfish** [comment]
Moved kotlin source here: https://github.com/herefishyfish/aptabase-kotlin for now before it gets pushed into the main org.

**Comment by herefishyfish** [comment]
We can either release this as is or wait for the official lib. Up to you guys.

**Comment by goenning** [comment]
@herefishyfish I was finally able to publish the library to maven central: https://central.sonatype.com/artifact/com.aptabase/aptabase/0.0.3

Jitpack seems abandoned and GitHub Packages requires auth even for public packages, so I decided to use maven central, It's a lot more work to setup, but I think it's easier for consumption.

I guess this means can remove the aar from this PR?

One small change though: I renamed the singleton from `shared` to `instance` as it seems like shared is more of a Swift naming convention, while Java/Kotlin seems to use instance more often

**Comment by herefishyfish** [comment]
Thanks @goenning 💯 

@NathanWalker we're using th official package now, should be good to publish 🚢

File: packages/nativescript-aptabase/index.android.ts [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
-        console.log('Not supported on Android yet.')
-    }
-}
\ No newline at end of file
+	static initialize(appKey: string, opts?: InitOptions) {
+		const initOptions = initOptionsFromOptions(opts);
+		com.aptabase.Aptabase.getInstance().initialize(Utils.android.getApplicationContext(), appKey, initOptions);
+	}
+
+	static track(name: string, properties?: any) {
```
Comment (goenning) [inline packages/nativescript-aptabase/index.android.ts:20]: tiny improvement here, we could replace `any` with `Record<string, string | number | boolean>`, that's what I use on aptabase-js

**Comment by NathanWalker** [comment]
Released with 1.0.5

**Pull request merged**